scenario/AddMaticxCollateralScenario.ts

```diff
@@ -0,0 +1,106 @@
+import { scenario } from './context/CometContext';
+import { expect } from 'chai';
+import { utils } from 'ethers';
+import { exp } from '../test/helpers';
+import { calldata } from '../src/deploy';
+import { impersonateAddress } from '../plugins/scenario/utils';
+import { createCrossChainProposal, matchesDeployment } from './utils';
+
+const MATICX_WHALES = {
+  polygon: ['0x68B9220B8E617b7700aCAE1a5Ff43F3eb29257F3'],
+};
+
+// TODO: add ability to run ad hoc scenarios against a single migration, to avoid needing the scenario to do all this setup of
+// listing an asset
+scenario(
+  'add new asset maticx',
+  {
+    filter: async (ctx) =>
+      matchesDeployment(ctx, [{ network: 'polygon' }]),
+    tokenBalances: {
+      $comet: { $base: '>= 1' },
+    },
+  },
+  async (
+    { comet, configurator, proxyAdmin, actors, bridgeReceiver },
+    context
+  ) => {
+    const { albert } = actors;
+    const dm = context.world.deploymentManager;
+    const maticx = await dm.existing(
+      'MATICX',
+      '0xfa68FB4628DFF1028CFEc22b4162FCcd0d45efb6',
```

Comment:
nit: maybe move this address into a constant

Reply:
sure